modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -497,6 +497,9 @@ public static class Network {
 
         /** Could not resolve address. */
         public static final int ADDRESS_UNRESOLVED_ERR = NETWORK_ERR_GROUP.registerErrorCode((short) 6);
+
+        /** Could not find nodes. */
+        public static final int NODE_FINDER_ERR = NETWORK_ERR_GROUP.registerErrorCode((short) 7);
```

Comment:
Error codes are a part of the public API, only exceptions that can be caught by users need to have such codes. But this code is used for exceptions that might only be observed by our code. I think the error code should not be introduced; instead, just use the INTERNAL_ERR code.